tweepy/api.py

```diff
@@ -242,6 +262,70 @@ def media_upload(self, filename, *args, **kwargs):
             upload_api=True
         )(*args, **kwargs)
 
+    def upload_chunked(self, filename, *args, **kwargs):
+        """ :reference https://developer.twitter.com/en/docs/media/upload-media/uploading-media/chunked-media-upload
+            :allowed_param:
+        """
+        f = kwargs.pop('file', None)
+
+        # Media category is dependant on whether media is attached to a tweet
+        # or to a direct message. Assume tweet by default.
+        is_direct_message = kwargs.pop('is_direct_message', False)
+
+        # Initialize upload (Twitter cannot handle videos > 15 MB)
+        headers, post_data, fp = API._chunk_media('init', filename, self.max_size_chunked, form_field='media', f=f, is_direct_message=is_direct_message)
```

Comment:
```suggestion
        headers, post_data, fp = API._chunk_media('init', filename, MAX_UPLOAD_SIZE_CHUNKED, form_field='media', f=f, is_direct_message=is_direct_message)
```

Reply:
I'm adding a separate constant, `MAX_CHUNKSIZE`. According to the docs, [the maximum chunk size is 5MB](https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/api-reference/post-media-upload-append), which the maximum total upload size for GIF and video [remains 15MB](https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/overview).